user/languages/ruby.md

```diff
@@ -310,44 +220,28 @@ engines, hosted service providers and so on.
 
 ### `$BUNDLE_GEMFILE` environment variable
 
-When `gemfile` is thus defined *and* the file exists in the repository,
+When `gemfile` is defined *and* a Gemfile file exists in the repository,
 we define the environment variable `$BUNDLE_GEMFILE`, which `bundle install`
 uses to resolve dependencies.
 
-If you need to work with multiple Gemfiles within a single job, you would
-need to override `$BUNDLE_GEMFILE` by passing `--gemfile=` flag:
+If you need to work with multiple Gemfiles within a single job, override
+`$BUNDLE_GEMFILE` by passing the `--gemfile=` flag:
 
 ```bash
 bundle install --gemfile=my_gemfile
 ```
 
-## Testing against multiple JDKs (JRuby)
-
-It is possible to test projects against multiple JDKs, namely
+## JRuby: Testing against multiple JDKs
 
-- OpenJDK 7
-- Oracle JDK 7
-- Oracle JDK 8
-- OpenJDK 6
-
-To do so, use the `jdk` key in your `.travis.yml`, for example:
+Test projects against multiple JDKs, by using the `jdk` key in your
+`.travis.yml`:
 
 ```yaml
 jdk:
   - oraclejdk7
```

Comment:
Given that we're removing the section about all 4, I'd add to this list `- oraclejdk8` .  What do you think?